test/message.test.js

```diff
@@ -43,17 +44,17 @@ describe('message', () => {
   });
 
   it('avatar should be set with provided user', () => {
-    message.user = user;
+    setUser(message);
     expect(avatar.getAttribute('name')).to.be.equal('Joan Doe');
     expect(avatar.getAttribute('abbr')).to.be.equal('JD');
     expect(avatar.getAttribute('img')).to.be.equal('/test/visual/avatars/avatar.jpg');
-    expect(avatar.getAttribute('title')).to.be.equal('Joan Doe (JD)');
+    expect(avatar.getAttribute('title')).to.be.equal('Joan Doe');
```

Comment:
Note: this assertion can be removed as it's responsibility of `vaadin-avatar` component to set its title.

Reply:
Done.